Docker/arch/arch-lts-vars.json

```diff
@@ -0,0 +1,14 @@
+{
+  "_comment": "Build with `packer build -var-file=arch-lts-vars.json arch.json`",
+  "docker_image": "saltstack/ci-arch:base",
```

Comment:
Will this still pull the correct docker image since we renamed the last one to -lts ?